Makefile

```diff
@@ -23,7 +23,23 @@ test: ## Run unit tests
 vet: ## Run go vet
 	@go vet $(addprefix ./, $(SOURCE_DIRS))
 
-check: fmtcheck vet lint build test ## Pre-flight checks before creating PR
+tidy: ## Tidy go dependencies
+	@go mod tidy
+
+check-license: $(SOURCES)
+	@./hack/check-license.sh "$(SOURCES)"
+
+fake:
```

Comment:
why we need the fake one?

Reply:
because I was experimenting and forgot to delete it :)